storio-sample-app/src/main/java/com/pushtorefresh/storio/sample/db/entities/Tweet.java

```diff
@@ -14,6 +17,7 @@
 // Which will generate type mapping code in compile time,
 // You just need to link it in your code.
 @StorIOSQLiteType(table = TweetsTable.TABLE)
+@StorIOContentResolverType(uri = TweetMeta.URI_STRING)
```

Comment:
Maybe. But I see Resolvers like com.pushtorefresh.storio.sample.provider.meta.TweetMeta#GET_RESOLVER and confused too. Why its not generated?